.github/workflows/compilation.yml

```diff
@@ -158,11 +158,11 @@ jobs:
     - name: Upload variants artifact ELF
       uses: actions/upload-artifact@v3
       with:
-        name: opl-debug-${{ steps.version.outputs.version }}-${{ matrix.docker }}
+        name: opl-debug-${{ steps.version.outputs.version }}
         path: opl-*.elf
 
   release:
-    needs: [build, build-variants, build-lang]
+    needs: [build, build-variants, build-debug, build-lang]
```

Comment:
This should not impact on performance since (IIRC) the critical path on the CI is the variants matrix

Reply:
Good